docs/index.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Redirecting to https://bob.github.io/repo/</title>
+<meta http-equiv="refresh" content="0; URL=0.7">
+<link rel="canonical" href="0.7">
```

Comment:
Woops, need to fix that title 😆 

Copied this from https://gist.github.com/domenic/1f286d415559b56d725bee51a62c24a7. I think it's the best we can do.

Reply:
Actually, I bet we can include this in a custom Helium extension :)

Reply:
We'll leave this here temporarily, since the jdk client is the only one that needs it so far, but I plan to integrate it into the next sbt-typelevel release.
- https://github.com/typelevel/sbt-typelevel/issues/199